packages/components/modules/messages/EditGroup/index.tsx

```
@@ -0,0 +1,129 @@
+
+  const onSubmit = handleSubmit((data: any) => {
+    const dirtyValues = filterDirtyValues({ values: data, dirtyFields })
+    const { image } = data
+    const uploadables: { image?: File | Blob } = {}
+    if ('image' in dirtyValues) {
+      if (image) {
+        uploadables.image = image
+      } else {
+        dirtyValues.deleteImage = true
+      }
+      delete dirtyValues.image
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)